src/Configuration/PlatformConfigMapper.spec.js

```diff
@@ -0,0 +1,17 @@
+import {
+  it, expect, describe, assert,
+} from 'vitest';
+import PlatformConfigMapper from './PlatformConfigMapper';
+
+describe('PlatformConfigMapper', () => {
+  it('should test if csv file is called ', async () => {
+    const csvparser = new PlatformConfigMapper('src/resource/test/item_mapping_config.csv');
```

Comment:
in testcases use simple non-prod used files

Reply:
updated